crates/wash-lib/src/cli/capture.rs

```diff
@@ -51,17 +50,10 @@ pub enum CaptureSubcommand {
 
 #[derive(Debug, Parser, Clone)]
 pub struct CaptureReplayCommand {
-    /// An actor ID to filter captured invocations by. This will filter anywhere the actor is the
-    /// source or the target of the invocation. If provided with an provider ID, it will filter down
-    /// to interactions only between the actor and provider
-    #[clap(name = "actor_id", long = "actor-id", value_parser)]
-    pub actor_id: Option<ModuleId>,
-
-    /// A provider ID to filter captured invocations by. This will filter anywhere the provider is
-    /// the source or the target of the invocation. If provided with an actor ID, it will filter
-    /// down to interactions only between the actor and provider
-    #[clap(name = "provider_id", long = "provider-id", value_parser)]
-    pub provider_id: Option<ServiceId>,
+    /// A component ID to filter captured invocations by. This will filter anywhere the component is the
+    /// source or the target of the invocation.
+    #[clap(name = "component_id", long = "component-id", value_parser)]
+    pub component_id: Option<String>,
```

Comment:
Should we have source and target ID so you can filter for either direction? I know the destination would be a little harder as you'd have to parse from the topic, but it doesn't feel too bad